pkg/virt-controller/services/template.go

```diff
@@ -319,6 +320,11 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 	var volumeMounts []k8sv1.VolumeMount
 	var imagePullSecrets []k8sv1.LocalObjectReference
 
+	// Need to run in privileged mode in Power or libvirt will fail to lock memory for VMI
```

Comment:
Can we just provide some CAPS to it, and does not set it as privileged?

Reply:
This one has been bothering me a bit, but I haven't been able to make it work otherwise, I tried playing around with capabilities as well as adjusting memory lock size but haven't been able to successfully make it work. I might be missing something so any nudge in the right direction is appreciated. 

The error I end up getting when running without privileged mode is the following: `internal error: Process exited prior to exec: libvirt:  error : cannot limit locked memory to 14680064: Operation not permitted'`

Reply:
Let's add this to a follow up which needs investigation.